src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -67,10 +68,18 @@ public static AddressBook getTypicalAddressBook() {
         for (Person person : getTypicalPersons()) {
             ab.addPerson(person);
         }
+
+        for (Tag tag : getTypicalTags()) {
+            ab.addTag(tag);
+        }
         return ab;
     }
 
     public static List<Person> getTypicalPersons() {
         return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
     }
+
+    public static List<Tag> getTypicalTags() {
+        return new ArrayList<>(Arrays.asList(new Tag("friends"), new Tag("owesMoney")));
+    }
```

Comment:
I think it would be better to save these tags as constants under `/testutil/TypicalTags.java`!

Reply:
Ah that's right. Got it!